client/src/pages/Ohm.js

```diff
@@ -24,11 +24,13 @@ const Heading = styled.h1`
 `;
 
 const OhmWrapper = styled.div`
-  background: url(${Ohm_background});
+  background-image: url(${OhmBackground});
+  background-repeat: no-repeat;
   color: var(--black);
   display: grid;
   grid-template: 100px 200px 200px / 375px;
   height: 844px;
+  width: 390;
```

Comment:
```suggestion
  width: 390px;
```
This is an oddly specific value 🤔